### PR TITLE
Fixed the website design and changed the name of few classes

### PR DESCRIPTION
**Pull request by ghost**
Worked on the design of few pages including login and signup pages and changed the names of the classes to make it easier to understand.

### CODE REVIEW

File: ashionlogin.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href = "https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fontawesome/5.15.3/css/all.min.css">
+    <title>Ashion</title>
```
Comment (khan8799) [inline ashionlogin.html]: change title name to **Fashion**

File: ashionlogin.html [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
+    <link rel="stylesheet" href = "https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css"">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"">
```
Comment (khan8799) [inline ashionlogin.html]: why downgrade the font-awesome's version?

File: ashionlogin.html [hunk 3]
```diff
@@ -15,63 +15,68 @@
...
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link active" aria-current="page" href="/index.html">HOME</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">WOMEN'S</a>
```
Comment (khan8799) [inline ashionlogin.html]: replace all href="#" to href="javascript:void(0)" or with actual file path

File: ashionlogin.html [hunk 4]
```diff
@@ -87,22 +92,22 @@ <h3 style="color: palevioletred;">Login</h3>
                     <input class="pswrd" type="password" placeholder="Enter Password" name="pswrd">
                     <div class="inner-container">
                         <label>
-                          <input type="checkbox" style="text-align: left;" id="remember" name="remember">Remember me
+                          <input type="checkbox" style="text-align: left;" id="remember-pswrd" name="remember">Remember me
```
Comment (khan8799) [inline ashionlogin.html]: do not use inline-css anywhere

File: ashionlogin.html [hunk 5]
```diff
@@ -87,22 +92,22 @@ <h3 style="color: palevioletred;">Login</h3>
...
                         </label>
-                        <a class = "forgot" style="text-decoration: none; color: gray;" href="">Forgot Password?</a>
+                        <a class = "forgot-pswrd" style="text-decoration: none; color: gray;" href="">Forgot Password?</a>
                     </div>
                     <br>
                     <button type="submit" style="background-color: bisque; width: 300px;">Login</button>
                     <br>
-                    <div id="or">-or-</div>
+                    <div id="or-divider">-or-</div>
```
Comment (khan8799) [inline ashionlogin.html]: id name should be in camelCase

File: ashionlogin.html (hunk 3 above)
Comment (ghost) [reply ashionlogin.html]: resolved

File: ashionlogin.html (hunk 4 above)
Comment (ghost) [reply ashionlogin.html]: resolved

File: ashionlogin.html (hunk 5 above)
Comment (ghost) [reply ashionlogin.html]: resolved

File: ashionlogin.html (hunk 2 above)
Comment (ghost) [reply ashionlogin.html]: Icons were disappearing while using the upgraded version but now the issue is resolved. Updated the website using icons from font awesome version 5.

**Review by khan8799 — APPROVED** [review]

**Pull request merged**